packs/rails/config.yml

```diff
@@ -0,0 +1,13 @@
+name: Rails from newrelic.com/integration
```

Comment:
This should probably be moved under the `ruby` umbrella folder

Reply:
moved 👍